react-native/storyly-react-native/StorylyDemo/package.json

```diff
@@ -11,7 +11,7 @@
   },
   "dependencies": {
     "react": "17.0.2",
-    "react-native": "0.67.3"
+    "react-native": "0.67.5"
```

Comment:
updated due to error with linking react-native dependencies in android gradle. for every version they have created a update with fix.
for reference: https://github.com/facebook/react-native/issues/35210